src/test/groovy/com/stormpath/tck/me/MeIT.groovy

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright 2016 Stormpath, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import com.jayway.restassured.http.ContentType
+import com.jayway.restassured.response.Cookie
+import com.jayway.restassured.response.Cookies
+import com.stormpath.tck.AbstractIT
+import com.stormpath.tck.util.*
+import com.stormpath.tck.responseSpecs.*
+import org.testng.annotations.BeforeClass
+import org.testng.annotations.Test
+
+import static com.jayway.restassured.RestAssured.*
+import static org.testng.Assert.*
+import static org.hamcrest.Matchers.*
+
+@Test
+class MeIT extends AbstractIT {
+    private TestAccount account = new TestAccount()
+    private String accessToken
+
+    @BeforeClass
+    private void getTestAccountAccessToken() throws Exception {
+        accessToken =
+            given()
+                .param("grant_type", "password")
+                .param("username", account.username)
+                .param("password", account.password)
+            .when()
+                .post(FrameworkConstants.OauthRoute)
+            .then()
+                .statusCode(200)
+                .contentType(ContentType.JSON)
+                .body("access_token", not(isEmptyOrNullString()))
+            .extract()
+                .path("access_token")
+        deleteOnClassTeardown(account.href)
+    }
+
+    @Test
```

Comment:
I think the value is precisely *less* maintenance work. Here's the scenario I had in mind:

The spec changes somewhere in the future, in a breaking way. Which tests need to be updated? What are they called?

My solution was to use the issues in the TCK repo as a granular representation of the spec's assertions, assumptions, and behavior, and each test would be linked to a particular issue(s).

The breaking-update workflow would then be:
- Spec PR defines where the spec has changed
- We find the TCK issues (via labels) that correspond to those spec section(s) and behavior(s)
- We find the tests (via javadoc comments) that correspond to those TCK issues

I already ran through this process once when we made the SPA/produces change yesterday.